### DISCUSSION TITLE
homing on a breakout board

### DISCUSSION THREAD

**Discussion by tcurdt** [grblHAL/core#643] [Q&A] [answered]
To test a few things I have a breakout board that runs grblHAL. Now I need homing - but unless I connect 3 limit switches and press them accordingly I (of course) get a

```
ALARM:9 (Homing fail. Could not find limit switch within search distances. Try increasing max travel, decreasing pull-off distance, or check wiring.)
```

Is there some kind of short cut for development/tinkering to convince grblHAL that home was reached without manually pressing some limit switches?

**Comment by terjeio** [comment, marked as answer]
Enable manual homing for all axes:
```
$22=33
$44=0
$45=0
$46=0
```
Then send:
`$H`
when in the home position.
